src/einsteinpy/plotting/geodesics/static.py

```diff
@@ -120,6 +125,31 @@ def _update(frame):
             self.fig, _update, frames=frames, interval=interval, blit=True
         )
 
+    def plot3d(self, geodesic, color="#{:06x}".format(random.randint(0, 0xFFFFFF))):
```

Comment:
Instead of this, can we pass a parameter, `use_3d = True` or something like that in the __init__ ?

Reply:
Okay, I'll add both features in the same plot function.